docs/rules/max-lines.md

```diff
@@ -0,0 +1,117 @@
+# enforce a maximum file length (max-lines)
+
+Some people consider large files a code smell. Large files tend to do a lot of things and can make following what's going on hard.
+
+## Rule Details
+
+This rule enforces a maximum number of lines per file, in order to aid in maintainability and reduce complexity.
+
+
+## Options
+
+This rule has a number or object option:
+
+* `"max"` (default `100`) enforces a maximum number of lines in a file
+
+* `"skipBlankLines": true` ignore lines made up purely of whitespace.
+
+* `"skipComment": true` ignore lines containing just comments
+
+### code
+
+Examples of **incorrect** code for this rule with a max value of `2`:
+
+```js
+/*eslint max-lines: ["error", 2]*/
+var a,
+    b,
+    c;
+```
+
+```js
+/*eslint max-lines: ["error", 2]*/
+
+var a,
+    b,c;
+```
+
+```js
+/*eslint max-lines: ["error", 2]*/
+// a comment
+var a,
+    b,c;
+```
+
+Examples of **correct** code for this rule with a max value of `2`:
+
+```js
+/*eslint max-lines: ["error", 2]*/
+var a,
+    b, c;
+```
+
+```js
+/*eslint max-lines: ["error", 2]*/
+
+var a, b, c;
+```
+
+```js
+/*eslint max-lines: ["error", 2]*/
+// a comment
+var a, b, c;
+```
+
+### skipBlankLines
+
+Examples of **incorrect** code for this rule with the `{ "skipBlankLines": true }` option:
+
+```js
+/*eslint max-lines: ["error", 2, {"skipBlankLines": true}]*/
+
+var a,
+    b,
+    c;
+```
+
+Examples of **correct** code for this rule with the `{ "skipBlankLines": true }` option:
+
+```js
+/*eslint max-lines: ["error", 2, {"skipBlankLines": true}]*/
+
+var a,
+    b, c;
+```
+
+### skipComments
+
+Examples of **incorrect** code for this rule with the `{ "skipComments": true }` option:
+
+```js
+/*eslint max-lines: ["error", 2, {"skipComments": true}]*/
+// a comment
+var a,
+    b,
+    c;
+```
+
+Examples of **correct** code for this rule with the `{ "skipComments": true }` option:
+
+```js
+/*eslint max-lines: ["error", 2, {"skipComments": true}]*/
+// a comment
+var a,
+    b, c;
+```
+
+## When Not To Use It
+
```

Comment:
👍  to ordinary link to the JSCS rule page, similar to Related Rules, except that under Compatibility one of the following precedes the link: JSLint, JSHint, JSCS

Good catch, I will submit a pull request to make the inconsistent ones compatible 😀